### DISCUSSION TITLE
Migration of "concatenated dataset" in MMpretrain

### DISCUSSION THREAD

**Discussion by EdyQiu** [open-mmlab/mmclassification#1516] [Q&A]
Hello, I want to test on multiple datasets, and I used to use "Concat dataset" in misclassification.
e.g. in config file:

   test=[
        dict(type='ImageNet',
        data_prefix="/home/dataset/robustness/imagenet_c_先损坏后提取特征/6400_original/brightness/1/",
        ann_file= animal_ann_file,
        pipeline= all_pipeline),    

        dict(type='ImageNet',
        data_prefix="/home/dataset/robustness/imagenet_c_先损坏后提取特征/6400_original/brightness/2/",
        ann_file= animal_ann_file,
        pipeline= all_pipeline),    

        dict(type='ImageNet',
        data_prefix="/home/dataset/robustness/imagenet_c_先损坏后提取特征/6400_original/brightness/3",
        ann_file= animal_ann_file,
        pipeline= all_pipeline)]

However, after I migrated files to MMpretrain, I found it hard to modify the config file to test on multiple datasets. 
Can you give me some suggestions on how to modify the config file? 
Many thanks!

**Comment by Ezra-Yu** [comment]
Can this tutorial help you? https://mmengine.readthedocs.io/en/latest/advanced_tutorials/basedataset.html#concatdataset

**Reply by EdyQiu** [reply]
Thanks! I used to search in the MMpretrain github repository, but the link does not work:
Thanks for pointing out the right place to find useful information!
![image](https://user-images.githubusercontent.com/23387293/234221679-244e6a13-fef6-460c-a176-6197e96d162d.png)
![image](https://user-images.githubusercontent.com/23387293/234221835-ef08a7f0-0757-4aae-9bc3-4eb0432b0d56.png)

**Reply by EdyQiu** [reply]
this is the link I refer to
https://github.com/open-mmlab/mmpretrain/blob/411e05a7057fa0bc189a60c1f2722d4012c60423/docs/en/user_guides/dataset_prepare.md

**Reply by Ezra-Yu** [reply]
I will fix it.

**Comment by EdyQiu** [comment]
![image](https://user-images.githubusercontent.com/23387293/234258351-e4b5206a-9ff4-424a-bdd4-c9c7899710cf.png)
This is how I solve the problem, but unlike misclassification, which prints out(and saves in .pkl file) the ACC of every dataset, 
this method gives ACC of the concat dataset. And the .pkl file has no overall information, which is not very convenient.
